de.dentrassi.pm.aspect.mvnosgi/src/de/dentrassi/pm/aspect/mvnosgi/internal/Constants.java

```diff
@@ -0,0 +1,10 @@
+package de.dentrassi.pm.aspect.mvnosgi.internal;
+
+public class Constants
+{
+    public static final String POM_FILE_NAME = "mvnosgi.pom";
```

Comment:
At the moment I used a static file name.
This is a virtual and should not conflict with a real file, so IMHO such a "stupid" name is ok.